requests/flash-attn-outputs.yml

```
@@ -0,0 +1,3 @@
+action: add_feedstock_output
+feedstock_to_output_mapping:
+  - flash-attn: "flash-attn*"
```

```suggestion
  - flash-attn: "flash-attn-"
```

This will keep the namespace more open.